source/options.tsx

```diff
@@ -22,7 +22,7 @@ function reportStatus({error, text, scopes}: Status): void {
 	if (error) {
 		tokenStatus.dataset.validation = 'invalid';
 	} else {
-		delete tokenStatus.dataset.validation;
+		tokenStatus.dataset.validation = 'valid';
```

Comment:
This is not equivalent. `valid` means valid, the lack of it means _undefined, we don't know yet._ It shows a different icon

Reply:
> * Restore valid checkmark in options

This isn't what I suggested